runtime/ftplugin/mbsync.vim

```diff
@@ -0,0 +1,19 @@
+" Vim filetype plugin file
+" Language:		mbsync configuration file
+" Maintainer:		Pierrick Guillaume <pguillaume@fymyte.com>
+" Last Change:		2025 Apr 12
+
+if (exists('b:did_ftplugin'))
+  finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo&vim
```

Comment:
You don't need this here, since you are not using line continuation in the filetype plugin. But you should add it to the syntax script instead.

Reply:
Done